zokrates_js/src/util.rs

```diff
@@ -26,3 +28,23 @@ pub fn normalize_path(path: PathBuf) -> PathBuf {
     }
     ret
 }
+
+pub fn get_seeded_rng(entropy: &str) -> StdRng {
```

Comment:
Would it make sense to put this in `zokrates_proof_systems` instead of duplicating it in `zokrates_js` and `zokrates_cli` ?

Reply:
If this happens, some dependencies can be removed